modules/core/src/v2/coins/trx.ts

```diff
@@ -20,6 +21,24 @@ import * as utxoLib from 'bitgo-utxo-lib';
 import { BitGo } from '../../bitgo';
 import { NodeCallback } from '../types';
 
+export interface TronSignTransactionOptions extends SignTransactionOptions {
+  txPrebuild: TransactionPrebuild;
+  prv: string;
+}
+
+export interface TransactionPrebuild {
+  txHex: string;
+  txid: string;
+  transaction: any;
```

Comment:
this is actually a string | { txID: string }

Reply:
also it can have signatures already in an array but its an optional param

Reply:
`transaction` is the full transaction object, not just the `id`
We need id so that `signTransaction` return the full transaction ready to be signed and broadcasted.